.travis.yml

```diff
@@ -1,10 +1,7 @@
 language: go
 
 go:
-  - 1.2
-  - 1.3
-  - 1.4
-  - tip
+  - 1.19
```

Comment:
This might be obvious but I didn't see any recent go version bump for this repo, is this needed?

Reply:
yes https://app.travis-ci.com/github/gametimesf/decimal/builds/255931389